src/client/elm/Transaction/Event.elm

```diff
@@ -294,6 +296,35 @@ type alias ProtocolUpdate =
     }
 
 
+type alias Description =
+    { name: String
+    , url: String
+    , description: String
+    }
+
+
+{-| Identification number of an anonymity revoker or identity provider
+-}
+type alias ArIpIdentity =
```

Comment:
I think it's a bad idea to have only the single type for two quite distinct concepts.

In particular ArIdentity cannot be 0, but IpIdentity can be 0. I guess this doesn't matter for displaying, but I feel that we should model it consistently in all parts of the system.